components/places/android/src/test/java/mozilla/appservices/places/PlacesConnectionTest.kt

```diff
@@ -382,15 +384,15 @@ class PlacesConnectionTest {
         assert(!PlacesManagerMetrics.writeQueryCount.testHasValue())
         assert(!PlacesManagerMetrics.writeQueryErrorCount["url_parse_failed"].testHasValue())
 
-        db.noteObservation(VisitObservation(url = "https://www.example.com/2a", visitType = VisitType.REDIRECT_TEMPORARY, at = 130000))
-        db.noteObservation(VisitObservation(url = "https://www.example.com/2b", visitType = VisitType.LINK, at = 150000))
-        db.noteObservation(VisitObservation(url = "https://www.example.com/3", visitType = VisitType.LINK, at = 200000))
+        db.noteObservation(VisitObservation(url = "https://www.example.com/2a", visitType = VisitTransition.REDIRECT_TEMPORARY, at = 130000u))
```

Comment:
while the move to unsigned looks correct, are we sure we want to take that on in this PR? It seems like this might have broad consequences in the a-c and fenix code? It might be possible to declare it in the udl as a signed int, and have `UniffiCustomTypeWrapper` do the conversion, and then maybe another ticket to fix it post-landing?